administrator/components/com_media/controllers/api.json.php

```diff
@@ -144,7 +144,7 @@ public function files()
 						$this->adapter->createFolder($name, $path);
 					}
 
-					$data = $this->adapter->getFiles($path . '/' . $name);
+					$data = $this->adapter->getFile($path . '/' . $name);
```

Comment:
But not in the current implementation if the api. The adapter calls create only one file/folder per request.

```
$this->adapter->createFile($name, $path, $mediaContent);

$this->adapter->createFolder($name, $path);
```